README.md

```diff
@@ -341,6 +342,21 @@ Notes about the above code:
     method name.
   * This attribute is not inherited and therefore must be explicitly set on any override.
 
+#### Workflow Task Scheduling
```

Comment:
Oh there is a ton to cover in the README, I just wanted to make this note so I didn't forget it.

Basically the README will look like Python's: https://github.com/temporalio/sdk-python#readme. There is gonna be a ton of stuff there coming in the next PR or two.

Specifically for testing workflows, the approach at https://github.com/temporalio/sdk-python#testing will also apply here. You will use `Temporalio.Testing.WorkflowEnvironment` class. These are actually more like integration tests since they run real servers as external processes, but they are fast, isolated, and give full functionality. The time-skipping test server is actually Graal-compiled from Java at https://github.com/temporalio/sdk-java that we use in TypeScript and Python today. In the future, if desired, we may offer a separate NuGet package with that test server as a DLL instead of lazy downloaded and run as a separate process.